tests/python/relay/test_op_level6.py

```diff
@@ -25,15 +25,14 @@
 
 @tvm.testing.uses_gpu
 def test_sort():
-    def verify_sort(shape, axis, is_ascend, is_dyn=False):
-
+    def verify_sort(shape, axis, is_ascend, is_dyn=False, in_dtype="float32"):
```

Comment:
We could also do a small refactoring for this file:
- Use `@pytest.mark.parametrize("in_dtype", ["float32", "float16"])` in each unit test.
- Let pytest collect tests in the main function:
```python
if __name__ == "__main__":
    pytest.main([__file__])
```

Reply:
Wouldn't that make the testing time double? This file is fairly slow to test (more than 3 min according to https://ci.tlcpack.ai/job/tvm/job/main/1384/testReport/ctypes.tests.python.relay/test_op_level6/) and I don't think fp16 tests need to run as often as fp32.

Reply:
Hmm that's a fair concern.